src/stock_str.rs

```diff
@@ -1262,6 +1239,24 @@ pub(crate) async fn chat_unencrypted_explanation(context: &Context) -> String {
     translated(context, StockMessage::ChatUnencryptedExplanation).await
 }
 
+impl Viewtype {
+    /// returns Localized name for message viewtype
+    pub async fn to_locale_string(&self, context: &Context) -> String {
+        match self {
+            Viewtype::Image => image(context).await,
+            Viewtype::Gif => gif(context).await,
+            Viewtype::Sticker => sticker(context).await,
+            Viewtype::Audio => audio(context).await,
+            Viewtype::Voice => voice_message(context).await,
+            Viewtype::Video => video(context).await,
+            Viewtype::File => file(context).await,
+            Viewtype::Webxdc => "Mini App".to_owned(),
+            Viewtype::Vcard => "👤".to_string(),
+            Viewtype::Unknown | Viewtype::Text | Viewtype::Call => self.to_string(),
```

Comment:
Maybe add a comment that for these an untranslated string shouldn't normally appear, otherwise may look as a lack of implementation